README.md

```diff
@@ -13,3 +13,22 @@ The *tutorials* folder contains walkthroughs which demonstrate basic Certifai fu
 ## Certifai Documentation
 
 Refer to the Cortex Certifai documentation: https://cognitivescale.github.io/cortex-certifai/docs/about for detailed information about Cortex Certifai.
+
+## Notebooks Table of Contents
+
+| Notebook Name | Description | Task Type | Model Types | Evaluation Types | Key Properties |
+| --- | --- | --- | --- | --- | --- |
+| [AnalyzeFeatureUsage.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/analyze_feature_usage) - Analyzing Feature Usage | Illustrates the use of Certifai to create and run an explanations scan of a locally defined model from first principles. It then analyzes and displays feature distribution of the counterfactuals.  |  Binary classification | Decision Tree <br /> Logistic Regression | Explanations |  Feature Occurrence <br /> Frequency by Model |
```

Comment:
'Feature occurrence frequency' is the metric its deriving, but the reason to do so is as a measure of feature importance.  'Feature importance' should be present as a key property